odxtools/database.py

```diff
@@ -39,9 +36,14 @@ def __init__(self,
         self._comparam_subsets = NamedItemList[ComparamSubset]()
         self._comparam_specs = NamedItemList[ComparamSpec]()
 
-    def add_pdx_file(self, pdx_file_name: str) -> None:
-        pdx_zip = ZipFile(pdx_file_name)
+    def add_pdx_file(self, pdx_file: Union[str, IO[bytes]]) -> None:
```

Comment:
nit: `ZipFile` works on any path-like object, i.e.,
```suggestion
    def add_pdx_file(self, pdx_file: Union[str, PathLike, IO[bytes]]) -> None:
```

(you need to add `from os import PathLike` above and should change the signatures of the other `add_*()` below...)

Reply:
Try to reflect the proposal in the new commit